js/background.js

```diff
@@ -101,31 +101,46 @@ function loginas(this_account, callback)
 	var site_info = get_config_site_info(this_account.site_info);
 	var conn = new XMLHttpRequest();
 	console.log("Beginning the login dance...");
-	getLJchallenge(site_info.interfaceurl, function (challenge) {
-		var response = md5(challenge + this_account.password);
-		var params = "mode=sessiongenerate" +
+	// As of 4/26/20 Dreamwidth no longer supports challenge/response with passwords
+	if (site_info.naem == "Dreammwidth") {

```

Comment:
naem?

Reply:
And 'Dreammwidth', because apparently I shouldn't write code before caffeine ;o;